workflow/scripts/infer_window_ts.py

```diff
@@ -0,0 +1,151 @@
+import ray
+import numpy as np
+import tsinfer
+import tskit
+import os
+import zarr
+
+############# this is the code to infer the tree sequence #############
+
+# Get parameters from snakemake
+window_size = snakemake.params.window_size
+output_dir = snakemake.params.output_dir
+window_type = snakemake.params.window_type
+report = snakemake.params.report
+input_vcf = snakemake.input.vcf
+
+# Load ancestral alleles
+ancestral_states = np.load(snakemake.params.ancestral_states)
```

Comment:
Do we need ancestral states passed in for any reason, do you think? We could just use a default character ('0' or something) -- I think this is what is done in Ben's tsinfer pipeline, for example

Reply:
Nevermind, I'm wrong about this (see below)